packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -32,46 +75,132 @@ export function FormatFile({ file, onDelete }: FormatFileProps) {
-      <div className={styles.imageBlock} style={style} data-testid="imageBlock">
+    <div
+      onMouseEnter={() => {
+        showDeleteButtonIfSmallThumbnail(
+          display === "compact" && displaySize === "default",
+          setShowDeleteButton,
+        );
+      }}
+      onMouseLeave={() => {
+        hideDeleteButtonIfSmallThumbnail(
+          display === "compact" && displaySize === "default",
+          setShowDeleteButton,
+        );
+      }}
+      className={
+        IsFileDisplay(display)
+          ? styles.formatFile
+          : thumbnailParentClassnames(imageSource, displaySize, isComplete)
+      }
+      style={
+        IsFileDisplay(display)
+          ? {}
+          : {
+              width: thumbnailDimensions.width,
+              height: thumbnailDimensions.height,
+            }
+      }
+    >
+      <div
+        className={imageBlockStyle}
+        style={
+          IsFileDisplay(display)
+            ? { ...style }
+            : {
+                ...style,
+                width: "inherit",
+                height: "inherit",
+              }
+        }
+        tabIndex={0}
+        data-testid="imageBlock"
+        onClick={event => {
+          onClick?.(event);
+          event.currentTarget.focus();
+        }}
+      >
         {!imageSource && (
-          <div className={styles.icon}>
-            <Icon name={iconName} />
-          </div>
-        )}
-        {!isComplete && (
-          <div className={styles.progress}>
-            <ProgressBar
-              size="small"
-              currentStep={file.progress * 100}
-              totalSteps={100}
-            />
-          </div>
+          <ImageWithoutSource
+            displayIsExpanded={IsFileDisplay(display)}
+            displaySize={displaySize}
+            iconName={iconName}
+            filename={file.name}
+          />
         )}
+        {!isComplete && <>{progressBar(file)}</>}
       </div>
-      <div className={styles.contentBlock}>
-        <Typography element="span">{file.name}</Typography>
-        <Typography element="p" size="small" textColor="greyBlueDark">
-          {fileSize}
-        </Typography>
-      </div>
-      {isComplete && onDelete && (
-        <div className={styles.actionBlock}>
-          <Button
-            onClick={onDelete}
-            type="tertiary"
-            variation="destructive"
-            icon="trash"
-            ariaLabel="Delete"
-          />
+      {IsFileDisplay(display) && (
+        <div className={styles.contentBlock}>
+          <Typography element="span">{file.name}</Typography>
+          <Typography element="p" size="small" textColor="greyBlueDark">
+            {fileSize}
+          </Typography>
         </div>
       )}
+      {isComplete && onDelete && showDeleteButton && (
+        <>
+          <FormatFileDeleteButton
+            deleteButtonStyle={
+              IsFileDisplay(display) ? styles.actionBlock : styles.deleteButton
```

Comment:
I think actionBlock and deleteButton might need to be renamed. The reason is that the Delete Button is not using the deleteButton style when it is a file. I think maybe it could be fileDeleteButton and imageDeleteButton maybe